install/kubernetes/helm/istio/charts/tracing/templates/NOTES.txt

```diff
@@ -13,7 +13,8 @@
   export SERVICE_IP=$(kubectl get services --namespace {{ .Release.Namespace }} zipkin -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
   echo http://$SERVICE_IP:{{ .Values.service.externalPort }}
 {{- else if contains "ClusterIP" .Values.service.type }}
-  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "zipkin.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app=zipkin,release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
Is that really supposed to be zipkin?

Reply:
Ah, nevermind. I see this is not replacing zipkin with jaeger, but an attempt to abstract tracing yet still provide both.

Reply:
Yes, it was an attempt to provide a zipkin service facade but support different providers. At the moment it is being superceded by #5670 as a more immediate solution for 0.8, but could form the basis of a longer term solution.